docker-images/perf-tuning/Dockerfile

```diff
@@ -3,7 +3,7 @@
 FROM nvidia/cuda:10.0-base
 
 RUN apt-get update && apt-get install -y python3.7 python3.7-dev libgomp1 cuda-cublas-10-0 python3-pip 
-RUN python3.7 -m pip install psutil gputil
+RUN python3.7 -m pip install psutil gputil numpy onnx packaging sympy
```

Comment:
Why are these modules needed? Are they for symbolic_shape_infer.py?

Reply:
Yes this is to simplify the symbolic shape inference to use the same docker, since Nuphar is not in pypi package yet.
```
docker run -v ~/data:/data --entrypoint python3.7 -t perf_tuning /perf_tuning/bin/RelWithDebInfo/all_eps/symbolic_shape_infer.py --input /data/path/to/model --output /data/path/to/shape_inferred_model --auto_merge
```

---
In reply to: [374969815](https://github.com/microsoft/OLive/pull/21#discussion_r374969815) [](ancestors = 374969815)